### PR TITLE
OCLOMRS-297: Fix wrong API URL endpoints

### PR DESCRIPTION
**Pull request by Duncanian**
# JIRA TICKET NAME:
[Fix wrong API URL endpoints](https://issues.openmrs.org/browse/OCLOMRS-297)

# Summary:
Some parts of the application are making API calls to the wrong API endpoint which is causing 404 error. These API endpoints URLs should be corrected to the right ones.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "jsx-a11y/label-has-associated-control": 0,
     "jsx-a11y/no-static-element-interactions": 0,
     "compilerOptions": { "target": "ES6", "module": "commonjs" },
-    "max-len": ["error", { "code": 100, "ignoreStrings": true }]
+    "max-len": ["error", { "code": 100, "ignoreStrings": true, "ignoreTemplateLiterals": true }]
```
Comment (dkayiwa) [inline .eslintrc.json]: Is this also related to the ticket?

Comment (Duncanian) [reply .eslintrc.json]: Yeah, it is allowing `TemplateLiterals` to be used and urls definitions are using `template literals`
